tsk/auto/tsk_auto.h

```diff
@@ -108,7 +108,12 @@ class TskAuto {
     void setFileSystemPassword(std::string fileSystemPassword) { m_fileSystemPassword = fileSystemPassword; }
 
     /**
-     * TskAuto calls this method before it processes the volume system that is found in an
+     * @returns A password that will be used when trying to open each file system or empty.
+     */
+    std::string getFileSystemPassword() { return m_fileSystemPassword; }
```

Comment:
Is there a reason you're not returning a `const std::string&` here and making the function `const`?

Reply:
I'm returning a `std::string` for consistency with other TSK string getters.  Adding `const` seems like a good idea that I missed.  Thanks for catching that.  It should be updated.

Reply:
It would be better to return `const std::string&` for all of those than to return a copy; the caller can always make a copy if one is needed.

Reply:
Sure.  I think it would be worth creating a separate issue/PR to address [those instances](https://github.com/search?q=repo%3Asleuthkit%2Fsleuthkit+%28path%3A*.cpp+OR+path%3A*.c+OR+path%3A*.hpp+OR+path%3A*.h%29+%22string+get%22&type=code&ref=advsearch) simultaneously for consistency in API.